ltops/utils.go

```diff
@@ -76,13 +76,13 @@ func GetLoadtestFileOrURL(reference string) ([]byte, error) {
 		logrus.Infof("resolved %s to %s", reference, tryURL)
 		return getURL(tryURL)
 	}
-
-	// TODO: Support a named branch. None of the active PRs seem to have their artifacts on S3.
-	// tryURL := fmt.Sprintf("https://releases.mattermost.com/loadtest-pr/%s/mattermost-load-test.tar.gz", reference)
-	// if err, data := getURL(tryURL); err == nil {
-	// 	logrus.Infof("resolved %s to %s", reference, tryURL)
-	// 	return nil, data
-	// }
+	// Support a named branch.
+	tryURL := fmt.Sprintf("https://releases.mattermost.com/loadtest-pr/%s/mattermost-load-test.tar.gz", reference)
+	logrus.Info(tryURL)
```

Comment:
Debugging?

Reply:
Not really. For some reason that functionality was commented out, but it works properly. I wanted to return it, since it quite useful

Reply:
Can we provide more info with the log if that's the case? Right now if I'm looking at the logs I'm going to have no idea what I'm seeing when this is logged

Reply:
@jwilander - I cleaned it up. Forgot to remove the Info line. Thanks.